mautic/mautic
# Fix send results to email - converts double and single quotes

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Yes
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/5866
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Based on https://github.com/mautic/mautic/issues/5866 
Add Converts double and single quotes  to html entity  decode tokend (https://www.w3schools.com/php/func_string_html_entity_decode.asp)

[//]: # ( As applicable: )
#### Steps to reproduce the bug:

Create a form with email and contact company

- Add action: send form results
![image](https://user-images.githubusercontent.com/462477/39662960-b3b288da-506a-11e8-9d74-0cbde26fd852.png)

- Add the token {formfield=company} in the subject
![image](https://user-images.githubusercontent.com/462477/39662961-c0350a24-506a-11e8-9f71-8614078889bc.png)

- Add an email address to send the results
- Save and close
- Fill the form. In the company field, write a company name with this symbol ' like: test'test
- Wait for the form result and check.
- See that in the subject it is written test'test
- See that in the content of the email, the company name is well written.

![image](https://user-images.githubusercontent.com/462477/39662963-c67e1d94-506a-11e8-87a8-cc43db85bca2.png)


#### Steps to test this PR:
1.  Repeat all steps.
2.  See subject is ok
